test/website-types/.vscode/extensions.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Let’s remove those files too from starter kit.

I don’t like that we are focusing on the single text editor here and suggestions is not very useful.